Features/LanguageAndSkills.feature

```diff
@@ -0,0 +1,98 @@
+Feature: Manage Languages and Skills on MARS Profile
+  As a user, I want to log in, add, edit, delete languages and skills so that I can showcase my expertise on my profile.
+
+  Background:
+    Given I am on the MARS login page
+
+  # Login Scenarios
+  Scenario: TC01 - Login with valid credentials
+    When I enter valid login credentials
+    And I click the login button
+    Then I should be redirected to the profile page
+
+  Scenario: TC02 - Login with invalid credentials
+    When I enter invalid login credentials
+    And I click the login button
+    Then an error message should be displayed for invalid credentials
```

Comment:
1.1/
Please use tables  and pass values to your scenarios for example look at the below scenario 
This way you will get exposure to use them  and will know how to pass test data to the tests
You all need to pass the values down from feature files to the steps see examples below


Example 

	Given I logged into the portal
	When I click on tab Education
	And I add my education 
	Then I am able to see my education'

Would  change to 


@Education
Scenario Outline: 1-Add education
	Given I logged into the portal
	When I click on tab Education
	And I add my education including '<Country>', '<University>', '<Title>', '<Degree>', '<Graduation Year>'
	Then I am able to see my education details including '<Country>', '<University>', '<Title>', '<Degree>', '<Graduation Year>'

	Examples:
	| Country     | University                      | Title  | Degree  | Graduation Year |
	| New Zealand | Whitireia Community Polytechnic | M.Tech | Diploma | 2021            |
	| Venezuela   | Hhutech University              | B.A    | Diploma | 2011            |

See  https://cucumber.io/docs/gherkin/reference/